dashboard/client/average_response_time/average_response_time.html

```diff
@@ -0,0 +1,6 @@
+<template name="averageResponseTime">
+  <h2>Average response time</h2>
+  <div id="average-response-time">
```

Comment:
Empty lines between block units improve readability. I am not saying that we need a space after every block element, but that we use empty lines in our HTML to improve readability.

For a parallel, think about reading a document. There are paragraph breaks and line breaks after header elements which give our eyes a rest and allow us to easily find important sections. We also use empty lines in our JS for similar reasons.